core/src/runtime.ts

```diff
@@ -14,19 +14,20 @@ export interface RegisteredPlugin {
 }
 
 export const createCapacitor = (win: WindowCapacitor): CapacitorInstance => {
+  const capCustomPlatform: CapacitorCustomPlatformInstance =
+    win.CapacitorCustomPlatform || null;
   const cap: CapacitorInstance = win.Capacitor || ({} as any);
   const Plugins = (cap.Plugins = cap.Plugins || ({} as any));
-  const capPlatforms: CapacitorPlatformsInstance = win.CapacitorPlatforms;
 
-  const defaultGetPlatform = () => getPlatformId(win);
-  const getPlatform =
-    capPlatforms?.currentPlatform?.getPlatform || defaultGetPlatform;
+  const getPlatform = () => {
+    return capCustomPlatform !== null
+      ? capCustomPlatform.name
+      : getPlatformId(win);
+  };
 
-  const defaultIsNativePlatform = () => getPlatformId(win) !== 'web';
-  const isNativePlatform =
-    capPlatforms?.currentPlatform?.isNativePlatform || defaultIsNativePlatform;
+  const isNativePlatform = () => getPlatformId(win) !== 'web';
```

Comment:
```suggestion
  const isNativePlatform = () => getPlatform() !== 'web';
```
I think this should use `getPlatform()`, right? `getPlatformId(win)` probably returns `web` in electron and that's why getPlatform return the `capCustomPlatform.name`

Reply:
I agree